packages/api/src/index.js

```diff
@@ -35,12 +37,13 @@ router.get('/', () => {
   )
 })
 
-router.all('*', () => new Response('Not Found.', { status: 404 }))
+router.all('*', () => new JSONResponse({ message: 'Not Found' }, { status: 404 }))
 
-async function serverError(error) {
-  return new Response(error.message || 'Server Error', {
-    status: error.status || 500,
-  })
+function serverError (error) {
+  console.error(error.stack)
```

Comment:
```suggestion
```

Reply:
do we want to keep this?

Reply:
i think is helpful to see errors in the log

Reply:
Yes keep, for my sanity!